dht_bootstrap.go

```diff
@@ -17,7 +17,7 @@ var DefaultBootstrapPeers []multiaddr.Multiaddr
 
 // Minimum number of peers in the routing table. If we drop below this and we
 // see a new peer, we trigger a bootstrap round.
-var minRTRefreshThreshold = 4
+var minRTRefreshThreshold = 10
```

Comment:
@aarshkshah1992 do you think there are reasons people may want to be able to configure this for themselves? No need to prematurely expose things, but when we start changing the constants around it's generally worth an evaluation

Reply:
This is too internal a value for users to want to/to be able to manipulate. We should not expose this as a functional option.
@Stebalien Do you have an opinion on this ?